airflow/www/utils.py

```diff
@@ -247,7 +247,7 @@ def action_logging(f):
     @functools.wraps(f)
     def wrapper(*args, **kwargs):
         # Only AnonymousUserMixin() does not have user attribute
```

Comment:
Should this comment be removed as well? Sounds like AnonymousUserMixin does have user attribute, but is None.

Reply:
You are right, on it.